CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+**1.4.0**
+* EON version: 1.4.0
+* SDK version: 0.12.0 (see [SDK changelog](https://github.com/HorizenOfficial/Sidechains-SDK/blob/0.12.0/CHANGELOG.md))
```

Comment:
if we want the link to be immediately working must be
blob/0.12.0-RC1/ 
instead of
blob/0.12.0/